src/Service.Tests/CosmosTests/MutationTests.cs

```diff
@@ -550,12 +550,12 @@ public async Task ValidateInheritanceOfReadPermissionFromAnonymous()
 }";
             GraphQLRuntimeOptions graphqlOptions = new(Enabled: true);
             RestRuntimeOptions restRuntimeOptions = new(Enabled: false);
-            Dictionary<string, JsonElement> dbOptions = new();
+            Dictionary<string, object> dbOptions = new();
```

Comment:
```suggestion
            Dictionary<string, object?> dbOptions = new();
```

Reply:
In this test method, we are not storing null values, so not required. Also it gives warning if i do so.